wrappers/core18_test.go

```diff
@@ -383,8 +396,12 @@ func (s *servicesTestSuite) TestAddSnapServicesForSnapdOnClassic(c *C) {
 
 	info := makeMockSnapdSnap(c)
 	// add the snapd service
-	err := wrappers.AddSnapdSnapServices(info, nil, progress.Null)
+	restart, err := wrappers.AddSnapdSnapServices(info, nil, progress.Null)
 	c.Assert(err, IsNil)
+	if restart != nil {
```

Comment:
shouldn't some of these check for restart being nil instead?  as for the classic case?

Reply:
I think you are right. I see only this one. The others are not classic.